QUICKSTART.md

```diff
@@ -81,5 +81,12 @@ The binding tools need the following to operate:
 
 In addition, the tools can operator on separate `.swiftmodule` and `.dylib` files. This can be handled by using a `-M` argument for the `.swiftmodule` and `-L` for the library. It’s easier to use the `.framework` directory and a single `-C` argument.
 
+## Building and Running Samples
+Samples source code is in the `samples` directory, but are will not build and run there.
```

Comment:
Think just a little typo, perhaps "... but will not build and run there."

Reply:
👍